src/main/java/com/bandwidth/iris/sdk/model/Site.java

```diff
@@ -40,7 +40,6 @@ public static Site create(IrisClient client, Site site) throws Exception {
     public static List<Site> list(IrisClient client) throws Exception {
         SitesResponse sitesResponse = client
                 .get(client.buildAccountModelUri(new String[] { IrisPath.SITES_URI_PATH }), SitesResponse.class);
-        client.checkResponse(sitesResponse);
```

Comment:
why are you removing this?

Reply:
I looked into the `client.get` and it does the same check.  It was redundant.

Reply:
It was redundant the `get()`  == 
```
public <T> T get(String uri, Class<T> returnType) throws Exception {
        IrisResponse response = get(uri);
        return processResponse(response, returnType);
    }
```
`processResponse()` contains this check on the return

```
 if (baseResponse.getResponseStatus() != null) {
                throw new IrisClientException(response.getStatusCode(), baseResponse.getResponseStatus().getErrorCode(),
                        baseResponse.getResponseStatus().getDescription());
            }
```

Which is all that the `checkResponse()` does

Reply:
ok cool.  Just not super familiar with this code.